client/app/components/land-use-action.js

```diff
@@ -94,4 +94,16 @@ export default class LandUseActionComponent extends Component {
     this.args.pasForm[actionToRemove.attr1] = '';
     this.args.pasForm[actionToRemove.attr2] = '';
   }
+
+  @action
+  reconstructLandUseActionSelections () {
+    const { pasForm } = this.args;
+    this.landUseActionOptions.map((option) => { //eslint-disable-line
```

Comment:
Also, this doesn't need to be an array `.map` because `.map` returns a new array. In this situation, it should be a `forEach`.

That said, I wonder if this could be broken out a little more into a separate getter. For example, you could have a getter: 

```javascript
get answeredQuestions() {
  const { pasForm } = this.args;

  return this.landUseActionOptions.filter(option => pasForm[option.field] || pasForm[option.attr1] || pasForm[option.attr2]);
}
```

That, in addition to the sorter getter, would be nice!

Reply:
Great suggestion!! Done!